components/raftstore/src/store/worker/split_check.rs

```diff
@@ -377,6 +383,17 @@ where
                 policy,
             } => self.check_split_and_bucket(&region, auto_split, policy),
             Task::ChangeConfig(c) => self.change_cfg(c),
+            Task::ApproximateBuckets(region) => {
+                if self.coprocessor.cfg.enable_region_bucket {
```

Comment:
If it's not enabled, the task should not be sent in the first place.

Reply:
Added a check before sending tasks.